packages/core/src/__tests__/__validation__/__schemas__/collection-reference.json

```
@@ -21,7 +21,7 @@
   "definitions": {
     "CeramicDocId": {
       "type": "string",
-      "pattern": "^ceramic://.+(\\?version=.+)?"
+      "pattern": "^ceramic://.+(\\?commit=.+)?"
```

I think it would be useful to discuss this change further, I don't think this concept of commit matters much to consumer beyond 3id-did-provider, while a concept of a version (specific contents) would.
I don't think commits should be a concern for the schemas we use.

---

Not sure why the query param is still left in the schema here. No longer needed as commits/versions are part of the DocID.
```suggestion
      "pattern": "^ceramic://.+"
```

---

We can discuss more, but the main reason `commit` might make more sense than `version` is that the concept of a version only really makes sense if there is one canonical log. In the future when we support CRDT doctypes that can have multiple tips, the concept of a version doesn't make much sense. 
The concept of a `commit` should be familiar to most developers, and I do think it makes sense for schemas as well.

---

I agree `commit` is familiar as in "I commit this change and I get a hash I can use as reference", but if I understand correctly commits would be performed by the anchor service, no?

---

Ah no. Commits would be all updates to a document. Signed records and anchor records.

---

See here https://github.com/ceramicnetwork/js-ceramic/pull/617 @PaulLeCam